blocks/carousel/carousel.css

```diff
@@ -178,10 +189,21 @@ main .section div.carousel-wrapper {
     border-radius: 50%;
 }
 
+.carousel.case-study .carousel-dots li {
+    width: 68px;
```

Comment:
I think the mobile width of the carousel dot/pager should be the same spacing value that we have in the design system instead of having a one-off `68px`. @sachinmesh Let's adjust the length of this to `64px` (aka `--spacer-layout-05`.

https://github.com/hlxsites/merative2/blob/main/styles/styles.css#L195